ntc_templates/templates/cisco_ios_ping.textfsm

```diff
@@ -0,0 +1,23 @@
+Value Required SENT_QTY (\d+)
+Value Required SENT_TYPE (.*)
+Value Required DESTINATION (\S+)
+Value Required TIMEOUT (\d+)
+Value SOURCE (\S+)
+Value Required RESPONSE_STREAM ([\.\!]+)
```

Comment:
this should probably be a List as sending something like 100 pings could wrap to a second line

Reply:
thank you for this...I did test things with 100 and 200, i probably should have a template for that to show, it did wrap visually on a second line but there was no line break so it still kept it in the variable...this could have also been due to how I was testing thing...
I will make this a list and add a test for that, thank you

Reply:
good call with the list
i added that, added a 200 ping test and adjusted the other tests
i don't suppose you know of a way to just run a single test?

Reply:
I think you would have to call pytest locally. https://docs.pytest.org/en/7.1.x/how-to/usage.html#specifying-which-tests-to-run